NitroxPatcher/Patches/CyclopsMotorModeButton_OnClick_Patch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Harmony;
```

Comment:
This file is new, but not referenced in the csproj file. It looks like this file came from PR #39, but you've changed some lines as well.

If this code is supposed to depend on another PR, do not copy the file but continue a new branch on top of that other branch, so that all the files and changes are present.